bcipy/gui/alert.py

```diff
@@ -0,0 +1,26 @@
+"""GUI alert messages"""
+# pylint: disable=no-name-in-module
+import sys
+from PyQt5.QtWidgets import QApplication, QMessageBox
+
+
+def confirm(message: str) -> bool:
```

Comment:
Some duplicate functionality and enums exist in [gui_main](https://github.com/CAMBI-tech/BciPy/blob/main/bcipy/gui/gui_main.py)

It may be useful to use the enums or factor this out of BCIGui to be used elsewhere.

Reply:
Good callout. I factored out the code for constructing a MessageBox into its own function in GUI/main that's now used in both BCIGui and here. They each call the resulting message box in slightly different ways and have separate use cases, so I think it still makes sense for them to be distinct, but now this module uses functionality provided by the main module.